integration-tests/neo4j/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+quarkus.neo4j.uri = ${neo4j.uri}
+quarkus.neo4j.authentication.username = ${neo4j.username}
+quarkus.neo4j.authentication.password = ${neo4j.password}
```

Comment:
We would need `quarkus.ssl.native=false` to get it to run properly on CI with native image.

Hopefully, it's gonna work without SSL?

Reply:
I now have a check in place. When native ssl is disabled (`!SslContextConfiguration.isSslNativeEnabled()`), than I turn off bolt encryption altogether. I added a note to the readme, though, because we really don't recommend that.

Worked in my tests, please check on CI.